### ISSUE TITLE
Changelog Commit appear at the latest Tag but was from an earlier Tag

### ISSUE THREAD

**Issue by Mom0aut** [tomasbjerre/git-changelog-lib#176]
Hey,

if i am generate a changelog sometimes a commit moves from a tag(release) to the latest tag

**Showcase expected**
```
Tag 0.0.10
- Commit 20
- Commit 19

Tag 0.0.2
- Commit 8
- Commit 7
```

**Showcase current**
```
Tag 0.0.10
- Commit 20
- Commit 19
- Commit 8
- Commit 7

Tag 0.0.2
- Empty
```
i use the goal `git-changelog-maven-plugin:git-changelog` with the example Changelog Template from the Git-Repo. In my real case the Tag/Commit Jumping appears over multiple Tags and it looks like the Commit happend the the latest Tag but they are not. Also the commits have often same names (Update from Gitflow or other Bumped Versioning Commits), could this maybe the problem?

Maybe i am using the maven goal wrong?

Thanks

**Comment by tomasbjerre** [comment]
Thanks for reporting, releasing a fix now. Open issue again if not working.

**Comment by tomasbjerre** [comment]
I released 1.100.7 of the Maven plugin. Should be available in Maven Central within a few minutes.
